### DISCUSSION TITLE
Why is there no mention of github discussions in nuxtjs.org?

### DISCUSSION THREAD

**Discussion by aptiko** [nuxt/nuxt#9800] [General]
Hi,

I was referred to github discussions some time ago by Pooya Parsa, whom I met in an unrelated meeting. Today I had forgotten where exactly this discussion place was and I had trouble finding it. I couldn't find any mention of it in nuxtjs.org. Eventually I remembered it had something to do with github, so I typed "github nuxt" in the browser's URL bar and it found the URL in its history.

Shouldn't GitHub Discussions be mentioned at nuxtjs.org?

**Comment by kissu** [comment]
I will talk about it tomorrow at Nuxtnation and we will clearly include more useful links with the new documentation, it needs some polish. 👍🏻

**Comment by atinux** [comment]
This is a good suggestion, do you mind opening an issue on nuxt/NuxtJS.org repo please?
